app/containers/ReactionsList.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { FlatList, StyleSheet, Text, Pressable, View } from 'react-native';
+import ScrollableTabView from 'react-native-scrollable-tab-view';
+
+import Emoji from './message/Emoji';
+import { useTheme } from '../theme';
+import { TGetCustomEmoji } from '../definitions/IEmoji';
+import { IReaction } from '../definitions';
+import Avatar from './Avatar';
+import sharedStyles from '../views/Styles';
+
+const styles = StyleSheet.create({
+	reactionsListContainer: { height: '100%', width: '100%' },
+	tabBarItem: {
+		paddingHorizontal: 10,
+		paddingBottom: 10,
+		justifyContent: 'center',
+		alignItems: 'center',
+		flexDirection: 'row'
+	},
+	reactionCount: { marginLeft: 5 },
+	tabLine: { position: 'absolute', left: 0, right: 0, bottom: 0 },
+	emojiName: { margin: 10 },
+	userItemContainer: { marginHorizontal: 10, marginVertical: 5, flexDirection: 'row' },
+	usernameContainer: { marginHorizontal: 10, justifyContent: 'center' },
+	usernameText: { fontSize: 17, ...sharedStyles.textMedium }
+});
+const standardEmojiStyle = { fontSize: 20, color: '#fff' };
```

Comment:
Why are these styles outside the stylesheet?

Reply:
Another thing, are you sure this prop accepts color as a parameter?

Reply:
Sorry that was by mistake.
Yes `color` is an acceptable prop as the standard emojis are rendered as text.